src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -1,14 +1,15 @@
 @import (reference) "../less/index";
 
 .NormalAnnouncementBubble--container {
+  max-width: 31rem; /* is 33 rem total width including 1 rem padding on either side */
   .boxShadow--medium();
-  border-radius: 1rem;
+  border-radius: 0.5rem;
   .padding--m();
   position: relative;
 }
 
 .NormalAnnouncementBubble--containerWithReply {
-  border-radius: 2rem;
+  border-radius: 1rem;
```

Comment:
These changes are specified in the [figma mocks](https://www.figma.com/file/HE5OffYDhp2nElIFahzYlt?node-id=430:22449#87383257)

Reply:
I think we talked about this before? Could use `@size_x` mixins here but don't have to

Reply:
Yes, you did fill me in on that previously. I stuck with the numbers because that's what was already there. I'm still feeling out what my preference is on these `@size` mixins.